x-pack/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -81,6 +81,8 @@ export const CLOUDBEAT_AZURE = 'cloudbeat/cis_azure';
 export const CLOUDBEAT_VULN_MGMT_AWS = 'cloudbeat/vuln_mgmt_aws';
 export const CLOUDBEAT_VULN_MGMT_GCP = 'cloudbeat/vuln_mgmt_gcp';
 export const CLOUDBEAT_VULN_MGMT_AZURE = 'cloudbeat/vuln_mgmt_azure';
+export const CIS_AWS = 'cis_aws';
+export const CIS_GCP = 'cis_gcp';
```

Comment:
note that `getBenchmarkIdIconType` in `CISBenchmarkIcon` is the only one using those values, it might be better to convert it to use the more commonly used `cloudbeat/cis_aws` convention which already exist.